localisation/000_Other_title_names.csv

```diff
@@ -367,4 +367,14 @@ b_menethil_adj;Lordain;;;;;;;;;;;;;x
 b_legation_quarter;Stanford;;;;;;;;;;;;;x
 b_legation_quarter_adj;Stanford;;;;;;;;;;;;;x
 b_great_cathedral;Anderton;;;;;;;;;;;;;x
-b_great_cathedral_adj;Anderton;;;;;;;;;;;;;x
\ No newline at end of file
+b_great_cathedral_adj;Anderton;;;;;;;;;;;;;x
+d_queldanas_isle;Quel'Danas Isle;Quel'Danas Isle;Quel'Danas Isle;;Quel'Danas Isle;;;;;;;;;x
```

Comment:
I think it's time to shorten it and call just **_Quel'Danas_**.

Reply:
agreed and done